package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "iconv-lite",
     "description": "Convert character encodings in pure javascript.",
-    "version": "0.4.24",
+    "version": "0.5.0",
```

Comment:
No problem taking the version change out -- I was just anticipating that if you accepted this changes, that's the version it would go into. My understanding of semantic version is that it the sequence is major-minor-bug, so the middle number is minor, and it gets bumped up for any added features, even a small ones, anything that's more than a bug fix.

As for the package-lock.json, that's certain easy enough to remove from the commit. Would you want package-lock.json to be in the .gitignore file?